rotkehlchen/tests/api/test_exchanges.py

```diff
@@ -410,25 +415,28 @@ def test_remove_exchange(rotkehlchen_api_server):
     # and check it's registered
     response = requests.get(api_url_for(rotkehlchen_api_server, 'exchangesresource'))
     result = assert_proper_response_with_result(response)
-    assert result == [{'location': 'coinbase', 'name': 'foo'}]
+    assert result == [{'location': 'coinbase', 'name': 'Coinbase 1'}]
 
     # Add query ranges to see that they also get deleted when removing the exchange
     cursor = db.conn.cursor()
     cursor.executemany(
         'INSERT OR REPLACE INTO used_query_ranges(name, start_ts, end_ts) VALUES (?, ?, ?)',
-        [('coinbasepro_trades', 0, 1579564096),
-         ('coinbasepro_margins', 0, 1579564096),
-         ('coinbasepro_asset_movements', 0, 1579564096),
-         ('coinbase_trades', 0, 1579564096),
-         ('coinbase_margins', 0, 1579564096),
-         ('coinbase_asset_movements', 0, 1579564096),
-         ('binance_trades', 0, 1579564096),
-         ('binance_margins', 0, 1579564096),
-         ('binance_asset_movements', 0, 1579564096)],
+        [('coinbasepro_trades_CoinbasePro 1', 0, 1579564096),
```

Comment:
note: we probably need to turn more of those tests into simple unit tests. To avoid having the api server setup which slows stuff down.